src/client/canvas/svg/arc.tsx

```diff
@@ -0,0 +1,67 @@
+import { observer } from 'mobx-react'
+import * as React from 'react'
+import { Component } from 'react'
+
+import { ArcGeometry } from '../geometry/arc_geometry'
+import { Shape } from '../object/shape'
+
+import { appearance } from './svg'
+
+@observer
+export class Arc extends Component<{model: Shape<ArcGeometry>}> {
+  // TODO draw arcs
+  public render() {
+    const m = this.props.model
+    const g = m.geometry
+    // g.anticlockwise
+    // g.startAngle
+    // g.endAngle
+    // g.origin
+    // g.radius
+    // rx ry x-axis-rotation large-arc-flag sweep-flag x y
+    return <path d={this.arc(g.origin,
+                             g.radius,
+                             g.startAngle,
+                             g.endAngle,
+                             g.anticlockwise)} {...appearance(m.appearance)} />
+  }
+
+
+  private arc(origin: {x: number, y: number},
+              r: number,
+              a0: number,
+              a1: number,
+              ccw: boolean): string {
+
+    // Is this arc empty?
+    if (r === 0) {
+      return ''
+    }
+    if (r < 0) {
+      throw new Error(`Negative radius: ${r}`)
+    }
+
+    // Work out our start and end points on the circle
+    const p0 = {
+      x: origin.x + r * Math.cos(a0),
+      y: origin.y + r * Math.sin(a0),
+    }
+
+    const p1 = {
+      x: origin.x + r * Math.cos(a1),
+      y: origin.y + r * Math.sin(a1),
+    }
```

Comment:
Can we use `Vector2` for this maths? And add a `Vector2.fromPolar(radius, angle)` static method to help this read nicely.